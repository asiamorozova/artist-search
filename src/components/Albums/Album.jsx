import React from 'react';
import PropTypes from 'prop-types';

const Album = ({ title, date }) => (
  <section>
    <h2>{title}</h2>
    <p>{date}</p>
  </section>
);

Album.propTypes = {
  id: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  date: PropTypes.string.isRequired
};

export default Album;

