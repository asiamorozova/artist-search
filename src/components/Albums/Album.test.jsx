import React from 'react';
import { shallow } from 'enzyme';
import Album from '../Albums/Album';

describe('Album', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Album
      id='string'
      title={'some song'} 
      date={'1969'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
