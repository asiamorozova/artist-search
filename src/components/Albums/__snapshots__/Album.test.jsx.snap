// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Album matches a snapshot 1`] = `
<section>
  <h2>
    some song
  </h2>
  <p>
    1969
  </p>
</section>
`;
